README.md

```diff
@@ -46,15 +46,19 @@ Install Janis dependencies.
 - $`npm install`
 
 Start a local environment with production data.
+- $`cp template.env prod.env` and set API_PASSWORD to production password
```

Comment:
I like the solution of having different .env files. It might be good to note that these are only used for local development, they aren't the source of truth for a deployed app. They're gitignored, but it might still be good to note.